build-system/babel-config/test-config.js

```diff
@@ -44,7 +44,10 @@ function getTestConfig() {
   const replacePlugin = getReplacePlugin();
   const replaceGlobalsPlugin = getReplaceGlobalsPlugin();
   const testPlugins = [
-    getImportResolverPlugin(),
+    getImportResolverPlugin({
+      'react': 'preact/compat',
+      'react-dom': 'preact/compat',
+    }),
```

Comment:
These config changes fix a compatibility issue between our tests and external libraries that depend on React.